lookout/style/typos/preparation.py

```diff
@@ -62,6 +62,7 @@ def update_to(self, b: int = 1, bsize: int = 1, tsize: Optional[int] = None) ->
 
 
 def _download_url(url: str, output_path: str) -> None:
+    os.makedirs(os.path.split(output_path)[0], exist_ok=True)
```

Comment:
Thank you, this needed to be done, but I suggest moving it to the beginning of `prepare_data()` function, as the same problem (directory doesn't exist) may also happen when the raw dataset is loaded from the local path.